guides/advanced_guides/authentication.md

```diff
@@ -1,34 +1,34 @@
-# Keys
+# Authentication
 
-In MeiliSearch, there are three types of keys:
+MeiliSearch uses a key-based authentication. There are three types of keys:
 
-- The **Master** key has access to all routes.
-- The **Private** key has access to all routes except the `/keys` routes.
-- The **Public** key has only access to the following routes:
+- The **Master** key grants access to all routes.
+- The **Private** key grants access to all routes except the `/keys` routes.
+- The **Public** key grants only access to the following routes:
   - `GET /indexes/:index_uid/search`
   - `GET /indexes/:index_uid/documents`
   - `GET /indexes/:index_uid/documents/:doc_id`
 
-When a master key is given to MeiliSearch, the engine will automatically generate the private and the public key. **You cannot create additional keys**.
+When a master key is provided to MeiliSearch, both the private and the public keys are automatically generated. **You cannot create any additional keys**.
 
 ### Master Key
 
-When launching an instance of MeiliSearch, you have the possibility to give a master key. By providing a master key, all routes in MeiliSearch will be protected and will require a key to access.
+When launching an instance, you have the option of giving a master key. By doing so, all routes will be protected and will require a key to be accessed.
 
-You can specify this key by passing the `MEILI_MASTER_KEY` environment variable, or with the command line argument `--master-key`.
+You can specify it by passing the `MEILI_MASTER_KEY` environment variable, or using the command line argument `--master-key`.
 
-With this master key, you can retrieve the private, and the public keys using the [keys route](/references/keys.md).
+Thanks to the master key, you can retrieve both the private and the public keys using the [keys route](/references/keys.md).
```

Comment:
Is thanks not used in an reassuring kind of way?
```suggestion
You can retrieve both the private and the public keys using the master key on the [keys route](/references/keys.md).
```

Reply:
Much better, thanks!